packages/redux/src/raffles/types/actions.types.ts

```diff
@@ -0,0 +1,194 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type {
+  BlackoutError,
+  Raffle,
+  RaffleEstimation,
+  RaffleParticipation,
+} from '@farfetch/blackout-client';
+import type { NormalizedSchema } from 'normalizr';
+
+/**
+ * Payload of a raffle
+ */
+type RaffleSuccessPayload = NormalizedSchema<
+  {
+    raffles: Record<Raffle['id'], Raffle>;
+  },
+  Raffle['id']
+>;
+
+/**
+ * Actions dispatched when the raffle state request is made.
+ */
+export interface FetchRaffleRequestAction extends Action {
+  type: typeof actionTypes.FETCH_RAFFLE_REQUEST;
+  meta: { raffleId: Raffle['id'] };
+}
+
+/**
+ * Actions dispatched when the raffle state success is made.
+ */
+export interface FetchRaffleSuccessAction extends Action {
+  type: typeof actionTypes.FETCH_RAFFLE_SUCCESS;
+  payload: RaffleSuccessPayload;
+  meta: { raffleId: Raffle['id'] };
+}
+
+/**
+ * Actions dispatched when the raffle state failure.
+ */
+export interface FetchRaffleFailureAction extends Action {
+  type: typeof actionTypes.FETCH_RAFFLE_FAILURE;
+  payload: { error: BlackoutError };
+  meta: { raffleId: Raffle['id'] };
+}
+
+export type FetchRaffleAction =
+  | FetchRaffleRequestAction
+  | FetchRaffleSuccessAction
+  | FetchRaffleFailureAction;
+
+/**
+ * Payload of a creating participation
+ */
+type RaffleParticipationSuccessPayload = NormalizedSchema<
+  {
+    raffleParticipations: Record<
+      RaffleParticipation['id'],
+      RaffleParticipation
```

Comment:
```suggestion
      RaffleParticipationEntity
```
Same for other payloads.